test/check-application

```diff
@@ -1951,7 +1951,7 @@ class TestFiles(testlib.MachineCase):
         b.wait_visible("[data-item='file0'].row-selected")
 
         # Test with very long hostnames
-        original_hostname = m.execute('hostnamectl hostname').strip()
+        original_hostname = m.execute('hostname').strip()
```

Comment:
Curious, surely rhel-8 has hostnamectl?

Reply:
Ah, `hostnamectl set-hostname hostname` should work, `hostnamectl hostname` was later added to hostnamectl as a convenience feature.

Reply:
Yes, exactly. See the [initial failure](https://cockpit-logs.us-east-1.linodeobjects.com/pull-917-9b03e008-20250127-222008-rhel-8-10-ws-container/log.html) for details (also linked from description and above)

Reply:
Right but we can use `hostnamectl set-hostname hostname`, anyway not too important

Reply:
No -- this doesn't *set* the host name, it *queries* it. `hostnamectl hostname` isn't a thing yet on the systemd version in RHEL 8 .